core/src/main/java/software/amazon/awssdk/core/http/pipeline/stages/HandleResponseStage.java

```diff
@@ -89,6 +89,10 @@ private OutputT handleSuccessResponse(HttpResponse httpResponse, RequestExecutio
         } catch (IOException | InterruptedException | RetryableException e) {
             throw e;
         } catch (Exception e) {
+            if (e instanceof SdkException && ((SdkException) e).retryable()) {
```

Comment:
could `successResponseHandler.handle` throw `SdkException`? I looked at some implementations but didn't find any.

Reply:
It could with S3. 

Maybe when using lambdas as well but not sure on that.